components/Billing/Billing.tsx

```diff
@@ -1043,10 +1043,12 @@ class BillingImpl extends JCComponent<Props, State> {
                                   paddingTop: 10,
                                   paddingBottom: 10,
                                   paddingLeft: 10,
-                                  paddingRight: 45,
+                                  paddingRight: 25,
                                 }}
                               >
-                                {line.description}
+                                {line.description.split("(")?.[0]?.split("×")[1]?.trim() ??
```

Comment:
Removes anything after the first open bracket, and removes anything before ×, removes trailing white spaces.
fallbacks in case no open bracket or × is found.